go.mod

```diff
@@ -4,16 +4,15 @@ go 1.13
 
 require (
 	github.com/golang/protobuf v1.3.1
-	github.com/hsanjuan/ipfs-lite v0.1.6
+	github.com/hsanjuan/ipfs-lite v0.1.7
 	github.com/ipfs/go-block-format v0.0.2
 	github.com/ipfs/go-cid v0.0.3
 	github.com/ipfs/go-ipld-cbor v0.0.3
 	github.com/ipfs/go-ipld-format v0.0.2
 	github.com/ipfs/go-log v0.0.1
 	github.com/ipfs/go-merkledag v0.2.3
-	github.com/libp2p/go-libp2p-crypto v0.1.0
```

Comment:
deprecated

Reply:
it's handy to run `go mod tidy` after `go get <new or updated package>`, which cleans out unneeded stuff from mod and sum files.

Reply:
do you mean this entire lib is deprecated, or this version is outdated?

Reply:
lib is deprecated in favor of go-libp2p-core/crypto

Reply:
ah I see, thx